### PR TITLE
Added config section to readme

### PR DESCRIPTION
**Pull request by rickdotnet**
Anyone looking to 'test' this out will need to add a config. This should help them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # PvpBot
 killl your discord friends
```
Comment (cxnky) [inline README.md]: Whilst you're editing this file, could you fix the spelling on this?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+
+
+# Config
+In order to run the bot, you'll need to add your token and prefix to a config file in your Bin directory.
+- Create a folder "bin\Resources"
+- Create a file called "config.json" and place it in the Resources folder
+- Add the following:
+
+```js
```
Comment (cxnky) [inline README.md]: This should be json, not js

File: README.md (hunk 1 above)
Comment (rickdotnet) [reply README.md]: Lol, unless he meant it to be that way, I'll submit another.

File: README.md (hunk 2 above)
Comment (rickdotnet) [reply README.md]: Good catch, I couldn't remember what the tag was. It shouldn't make a difference though.,

**Pull request closed without merging**